components/portal/messages/partials/notifications-bell.html

```diff
@@ -114,8 +114,11 @@
          ng-if="vm.priorityNotifications.length == 1"
          ng-repeat="priority in vm.priorityNotifications | limitTo: 1"
          layout="row" layout-align="center center" layout-fill>
-      <div flex layout="row" layout-align="center center" class="compact-buttons">
-        <span class="notification-message">{{ priority.title }}</span>
+      <div class="notification-message">
+        <span>{{ priority.title | limitTo: vm.titleLengthLimit | trim }}</span>
+        <span ng-if="priority.title.length > vm.titleLengthLimit">...</span>
```

Comment:
Clever. Possibly easier to read than a filter that does the limiting and trimming and `...`ing all in one fell swoop?